### ISSUE TITLE
About Build_dataset and NaN Loss

### ISSUE THREAD

**Issue by foureye** [D10Andy/SAD#4]
Q1:"mooc data, rand_feat = np.zeros((max_idx + 1, 172)), 172 need change to 4",I found shape of edge_feature and node_feature will be [12344,4],[4849,4]，then in feat_process_layer，i meet RuntimeError： mat1 and mat2 shapes cannot be multiplied (4849x4 and 172x128)。Do i need to change the dim of feat_process_layer when processing mooc?

Q2: I meet NaN in loss when training，”if torch.any(torch.isnan(log_probs)): raise ValueError("Log_prob has nan!")“
      I change learning rate from 5e-4 to 1e-6 solve the problem in wikipedia，but I still meet NaN in reddit dataset. 
     Do I need to keep turn dowm learning rate,or check logits/log_pros/sim_coeff?

Thank you!

**Comment by 000096** [comment]
请问您第一个问题解决了么
